tests/core/test_utils.py

```diff
@@ -2,6 +2,7 @@
 from __future__ import unicode_literals
 import unittest
 import os
+import pytest
```

Comment:
Oh, I'm really sorry. I didn't get that. :/

But why is pytest.raises better than assertRaises?